ingestion/setup.py

```diff
@@ -108,7 +108,7 @@ def get_long_description():
     "data-profiler": {"openmetadata-data-profiler"},
     "snowflake": {"snowflake-sqlalchemy<=1.2.4"},
     "snowflake-usage": {"snowflake-sqlalchemy<=1.2.4"},
-    "sample-data": {"faker~=8.1.1"},
+    "sample-entity": {"faker~=8.1.1"},
```

Comment:
changing this name here means we need to change the documentation. sample-data has been registered by our users , do we want to change this?